### PR TITLE
Update of HLT menu (80X)

### PR DESCRIPTION
**Pull request by silviodonato**
This PR updates the HLT menus, especially the pPb that has been almost completely developed in the last weeks.

The menus are based on the master table /dev/CMSSW_8_0_0/HLT/V660.

The change of subtables.sh is a fix of the previous PR. 
cmsDriver.csh has been changed to be working also outside CERN.

### CODE REVIEW

**Comment by silviodonato** [comment]
please test

**Comment by Martin-Grunewald** [comment]
errors not related to this PR
```
----- Begin Fatal Exception 09-Nov-2016 18:50:14 CET-----------------------
An exception of category 'DatabaseError ' occurred while
   [0] Processing run: 160960
   [1] Calling global beginRun for module LumiProducer/'lumiProducer'
Exception Message:
Can not get data (Additional Information: [frontier.c:1124]: No more proxies. Last error was: Request 83 on chan 2 failed at Wed Nov  9 18:50:14 2016: -8 [payload.c:101]: Server signalled payload error 1: LumiProd java.sql.SQLException: ORA-00942: table or view does not exist at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:456)) ( CORAL : "coral::FrontierAccess::Statement::execute" from "CORAL/RelationalPlugins/frontier" )
----- End Fatal Exception -------------------------------------------------
```

**Comment by Martin-Grunewald** [comment]
please test

**Comment by silviodonato** [comment]
+1

**Comment by mmusich** [comment]
@silviodonato @Martin-Grunewald 
please merge https://github.com/cms-tsg-storm/cmssw/pull/17
it adresses the issues reported at: https://github.com/cms-sw/cmssw/pull/16606#issuecomment-262253173

**Comment by Martin-Grunewald** [comment]
please test

File: Configuration/HLT/python/autoCondHLT.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
     '25ns15e33_v4' : ( ','.join( [ 'L1Menu_Collisions2016_v6r5_ugt_1board_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-08-23 20:00:00.000"] ), ),
     '25ns10e33_v2' : ( ','.join( [ 'L1Menu_Collisions2016_v4_xml'                            ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-05-24 14:55:00.000"] ), ),
     'HIon'         : ( ','.join( [ 'L1Menu_CollisionsHeavyIons2015_v5_uGT_xml'               ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-03-04 15:00:00.000"] ), ),
-    'PIon'         : ( ','.join( [ 'L1Menu_HeavyIons2016_v0_xml'                             ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-08-19 08:40:00.000"] ), ),
+    'PIon'         : ( ','.join( [ 'L1Menu_HeavyIons2016_v1_m2_xml'                          ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-10-28 16:09:24.000"] ), ),
```
Comment (Martin-Grunewald) [inline Configuration/HLT/python/autoCondHLT.py]: Fixed - thanks!

**Comment by Martin-Grunewald** [comment]
please test

**Comment by mmusich** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**